src/VirtualClient/VirtualClient.Core/Proxy/ProxyBlobManager.cs

```diff
@@ -88,11 +88,9 @@ public async Task<DependencyDescriptor> DownloadBlobAsync(DependencyDescriptor d
                 blobDescriptor.ContentEncoding.WebName,
                 blobPath);
 
-            HttpResponseMessage response = await this.ApiClient.DownloadBlobAsync(info, downloadStream, cancellationToken)
+            await this.ApiClient.DownloadBlobAsync(info, downloadStream, cancellationToken)
                 .ConfigureAwait(true);
 
-            response.ThrowOnError<DependencyException>(ErrorReason.DependencyInstallationFailed);
```

Comment:
Is there a reason we are taking the error reporting out? Did you move this responsibility in the apiclient? It's not clear to me that you are letting client reporting all errors

Reply:
added back in